libindy/src/api/mod.rs

```diff
@@ -116,6 +116,9 @@ pub enum ErrorCode
     // Returned if provided wallet query is invalid
     WalletQueryError = 214,
 
+    // Returned if wallet it not closed, which is required for the operation (currently for deletion)
+    WalletNotClosed = 215,
```

Comment:
Could we just change the error to CommonInvalidState? It will correspond to behavior with deletion of pool and we will not need to update all wrappers and cli

Reply:
Sure no problem